lib/content/routes.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+/**
+ * @fileoverview Content system routing functions
+ *
+ */
+const config = require('config');
+const types = require('punchcard-content-types');
+const _ = require('lodash');
+
+const database = require('../database');
+const utils = require('../utils');
+
+/**
+ * All Content-Types Landing page
+ * @param {object} res - HTTP Response
+ * @param {object} allTypes all content types available to the CMS
+ */
+const all = (res, allTypes) => {
+  res.render('content/home', {
+    content: {
+      home: config.content.home,
+      base: config.content.base,
+      types: allTypes,
+    },
+  });
+};
+
+/**
+ * Individual Content-Types Landing page
+ * @param {object} res - HTTP Response
+ * @param {object} type - Content type configuration, after merged with input plugins
+ *
+ * @returns {promise} database promise returned
+ */
+const one = (res, type) => {
+  return database
+    .distinct(database.raw('ON (id) id'))
+    .select('*')
+    .from(`content-type--${type.id}`)
+    .orderBy('id', 'DESC')
+    .orderBy('revision', 'DESC').then(rws => {
+      // add itentifier to each row
+      const rows = utils.routes.identifier(rws, type);
```

Comment:
I'm thinking we should update the database model to have a separate field for identifier so we don't need to run identifier over every DB call to get the identifier